build_library/toolchain_util.sh

```diff
@@ -142,15 +142,25 @@ get_board_binhost() {
     board="$1"
     shift
 
+    local no_toolchain=0
     if [[ $# -eq 0 ]]; then
-        set -- "${FLATCAR_VERSION_ID}"
+        if [[ "${FLATCAR_BUILD_ID}" =~ ^nightly-.*$ ]] ; then
+            # containerised nightly build; this uses [VERSION]-[BUILD_ID] for binpkg url
+            #  and toolchain packages are at the same location as OS image ones
+            set -- "${FLATCAR_VERSION_ID}-${FLATCAR_BUILD_ID}"
+            no_toolchain=1
```

Comment:
Should this set `toolchain_only` variable back to `0`, or should it bail out when `toolchain_only` is `1`? Having both `no_toolchain` and `toolchain_only` set to `1` sounds contradictory.

Reply:
The case we are looking at provides the toolchain packages at the same source as the OS image ("board") packages, so we skip adding a separate URL for toolchain packages later.

Reply:
I've updated the variable name to be less confusing.